runtime-modules/bounty/src/benchmarking.rs

```diff
@@ -134,32 +246,37 @@ benchmarks! {
             work_period: One::one(),
             judging_period: One::one(),
             cherry,
-            creator: BountyCreator::Member(member_id),
+            entrant_stake,
+            creator: BountyActor::Member(member_id),
+            funding_type: FundingType::Perpetual{ target: max_amount },
             ..Default::default()
         };
```

Comment:
I think this should also have a `Closed` contract type since it's the worst case scenario for `ensure_create_bounty_parameters_valid`.

Reply:
OK.